### PR TITLE
Update / Fix roles

### PR DESCRIPTION
**Pull request by IonesioJunior**

### CODE REVIEW

File: packages/grid/backend/grid/api/roles/roles.py [hunk 1]
```diff
@@ -159,13 +161,14 @@ def update_use_route(
...
-    can_manage_infrastructure: bool = Body(..., example="false"),
-    can_upload_data: bool = Body(..., example="false"),
+    can_make_data_requests: bool = Body(False, example="false"),
+    can_triage_data_requests: bool = Body(False, example="false"),
+    can_manage_privacy_budget: bool = Body(False, example="false"),
+    can_create_users: bool = Body(False, example="false"),
+    can_manage_users: bool = Body(False, example="false"),
+    can_edit_roles: bool = Body(False, example="false"),
+    can_manage_infrastructure: bool = Body(False, example="false"),
+    can_upload_data: bool = Body(False, example="false"),
```
Comment (tcp) [inline packages/grid/backend/grid/api/roles/roles.py:171]: Missing "Can Upload Legal Documents"

Comment (tcp) [inline packages/grid/backend/grid/api/roles/roles.py:171]: Missing "Can Edit Domain Settings"

**Review by tcp — CHANGES_REQUESTED** [review]
Good stuff removing groups Ionesio!

There are 2 permissions that are still missing: `Can Upload Legal Documents` and `Can Edit Domain Settings`.

File: packages/grid/backend/grid/api/roles/roles.py (hunk 1 above)
Comment (IonesioJunior) [reply packages/grid/backend/grid/api/roles/roles.py:175]: Fixed by 4d75037

**Review by tcp — APPROVED** [review]

**Pull request merged**